app/helpers/kaui/date_helper.rb

```diff
@@ -13,7 +13,7 @@ def format_date(date, timezone="Pacific Time (US & Canada)")
 
       # If not, convert into account timezone and return the date part only
       parsed_date = DateTime.parse(date.to_s).in_time_zone(timezone)
-      parsed_date.to_s(:date_only)
+      parsed_date.to_s(:pretty)
```

Comment:
Thank you Pierre. Timezone could be annoying thing. I'll create email about requirements of time for customer support this morning (PDT). 

Just one last question, for formatting date & time, 
I think we need 2 things: 
1. modify page, adding sth like `<p><%= l Time.now, format: :short %></p>`
2. make sure that data (`Time.now`) support it, right ? 
[transaction_date = Date.parse(transaction.effective_date)](https://github.com/killbill/killbill-admin-ui/blob/8bb930ecbdbe5bcefe568cb3f7c37ba914e8504f/app/controllers/kaui/payments_controller.rb#L41
) 
[format_date(transaction.effective_date, account.time_zone).html_safe ](https://github.com/killbill/killbill-admin-ui/blob/c59c681fd6f1ecb9e1eaef60b040411946a52922/app/views/kaui/payments/_payment_table.html.erb#L84)
these only have date, like '2017-02-11', but no part of time, it still shows date only with I18n ?